core/controllers/editor_test.py

```diff
@@ -1928,3 +1928,26 @@ def test_discard_draft(self):
         self.assertIsNone(exp_user_data.draft_change_list)
         self.assertIsNone(exp_user_data.draft_change_list_last_updated)
         self.assertIsNone(exp_user_data.draft_change_list_exp_version)
+
+
+class EditorFirstTimeTutorialIntegrationTest(BaseEditorControllerTests):
+    """This controller does integration tests of
+    the first time tutorial instances for the exploration editor.
```

Comment:
Can we move some text to the line above (As it looks a bit weird to me :p)? (following 80 chars per line.)